### PR TITLE
added fetch resources details for SCP

### PR DESCRIPTION
**Pull request by adithyadinesh0412**

### CODE REVIEW

File: src/constants/config.json [hunk 1]
```diff
@@ -7017,6 +7017,41 @@
 					"packageName": "elevate-self-creation-portal"
 				}
 			]
+		},
+		{
+			"sourceRoute": "/interface/v1/internal/fetchResources",
```
Comment (priyanka-TL) [inline src/constants/config.json:7022]: @adithyadinesh0412  this can be something like getAllPublishedResources

File: src/constants/config.json [hunk 2]
```diff
@@ -7017,6 +7017,41 @@
...
+							"sourceField": "data",
+							"targetField": "data[]"
+						}
+					]
+				},
+				{
+					"basePackageName": "samiksha",
+					"packageName": "elevate-survey",
+					"targetBody": [],
+					"responseBody": [
```
Comment (priyanka-TL) [inline src/constants/config.json:7047]: @adithyadinesh0412  why we need to do this "targetField": "data[]"

**Review by priyanka-TL — CHANGES_REQUESTED** [review]

**Review by priyanka-TL — CHANGES_REQUESTED** [review]
Reviewed on Aug 22nd  5.56 PM

File: src/constants/config.json (hunk 2 above)
Comment (adithyadinesh0412) [reply src/constants/config.json:7047]: This is the method to combine the results in the interface service by design. @priyanka-TL

**Review by priyanka-TL — APPROVED** [review]

File: src/constants/config.json [hunk 3]
```diff
@@ -7017,6 +7017,55 @@
 					"packageName": "elevate-self-creation-portal"
 				}
 			]
+		},
+		{
+			"sourceRoute": "/scp/v1/resource/browseExisting",
```
Comment (aks30) [inline src/constants/config.json:7022]: @adithyadinesh0412 let's call it resource/listing instead of browse existing.

**Review by aks30 — CHANGES_REQUESTED** [review]
Reviewed on 24 Aug

**Comment by adithyadinesh0412** [comment]
merging this PR to unblock FE

**Pull request merged**